doc/api/util.md

```diff
@@ -678,13 +678,71 @@ The default styles and associated colors are:
 * `symbol`: `green`
 * `undefined`: `grey`
 
-The predefined color codes are: `white`, `grey`, `black`, `blue`, `cyan`,
-`green`, `magenta`, `red` and `yellow`. There are also `bold`, `italic`,
-`underline` and `inverse` codes.
-
 Color styling uses ANSI control codes that may not be supported on all
 terminals. To verify color support use [`tty.hasColors()`][].
 
+The predefined color codes are:
```

Comment:
Seems like this line should be removed now?

Reply:
I guess we could remove it. I thought it makes it clearer to the reader though, that we have these types of predefined color codes. Should I remove this line?

Reply:
I am not sure where exactly you would move it too in this case?

Right now that's exactly the case: the color codes are divided into three parts. The `modifiers`, `colors` and `background colors`. The part above are the styles.

Reply:
I would expect "predefined color codes" to be the stuff under `Colors` rather than `Modifiers`. If this is the best layout in your opinion, then I'm OK with it. I would be inclined to either move this below Modifiers and above Colors, or else move Modifiers to be after Colors (and maybe after Background Colors too).

Reply:
I just pushed another commit with which I try to make this clearer. PTAl :-)